runtime/doc/usr_40.txt

```diff
@@ -290,8 +290,8 @@ deletes the first line.
 
 	Note:
 	User-defined commands must start with a capital letter.  You cannot
-	use ":X", ":Next" and ":Print".  The underscore cannot be used!  You
-	can use digits, but this is discouraged.
+	use ":Next".  The underscore cannot be used!  You can use digits, but
```

Comment:
Yes, the command itself was removed a while ago, but this piece of documentation must have been missed.